iOS/Flows/Conversation/Cells/ConversationMessagesCell.swift

```diff
@@ -31,6 +31,9 @@ class ConversationMessagesCell: UICollectionViewCell {
     }()
     private lazy var dataSource = MessageSequenceCollectionViewDataSource(collectionView: self.collectionView)
 
+    /// If true we should scroll to the last item in the collection in layout subviews.
+    private var scrollToLastItemOnLayout: Bool = false
```

Comment:
Getting the layout object to properly scroll to the last item on start is inconsistent. I'm moving back to having the cell handle the scrolling behavior, but I think it's written a little more cleanly this time.